packages/xo-server-test/src/_xoConnection.js

```diff
@@ -155,8 +181,10 @@ const getConnection = credentials => {
 let xo
 beforeAll(async () => {
   xo = await getConnection()
+  await xo.createTempServer(config.servers.default)
```

Comment:
I'm going to do the same thing for the remotes in another PR. Do I have to go back to the previous model (with `preCreatedResources` and `ARGS_BY_TYPE`)?

Reply:
I think this [comment](https://github.com/vatesfr/xen-orchestra/pull/4381#discussion_r309623194) respond to your question